src/coreclr/vm/precode.h

```diff
@@ -76,9 +76,9 @@ struct InvalidPrecode
 
 struct StubPrecodeData
 {
-    PTR_MethodDesc MethodDesc;
+    TADDR SecretParam;
     PCODE Target;
-    BYTE Type;
+    TADDR Type;
```

Comment:
Do we need TADDR here? It doesn't matter from the size perspective, but I was thinking we might possibly find some use for the remaining 3 bytes. But I am fine with having TADDR there.